engines/csv.py

```diff
@@ -61,6 +61,7 @@ def format_insert_value(self, value, datatype):
         return v
 
     def insert_statement(self, values):
+        """Returns a SQL statement to insert a set of values."""
```

Comment:
Since this is in the csv engine it actually "Returns a comma delimited row of values".